firebase-project/src/views/postList.vue

```diff
@@ -131,12 +233,128 @@ export default {
 
     getAllPosts();
 
+    const goBackPost = () => {
+      router.push("/post");
+    };
+
+    const showCreateButton = computed(() => {
```

Comment:
what or which create button @mukeshsin 
Try to take variable name good

Reply:
done